src/test/proxy/server_proxy.py

```diff
@@ -0,0 +1,44 @@
+from ja.user.proxy import IUserServerProxy
+from ja.server.database.database import ServerDatabase
+from ja.common.message.server import ServerResponse
+from ja.common.job import Job, JobStatus
+
+from ja.user.config.add import AddCommandConfig
+from ja.user.config.cancel import CancelCommandConfig
+from ja.user.config.query import QueryCommandConfig
+
+
+class ServerProxyDummy(IUserServerProxy):
+
+    def __init__(self, db: ServerDatabase)
```

Comment:
I think that the dummy for ServerProxy should not depend on ServerDatabase.
The point of the dummy is to implement and test the other components of the user client without needing an implementation of the server or its components, right?
@ammen99 @nikolatzotchev what do you think?

Reply:
Yes, dummy is supposed to be something simple, that is used for testing other components. The database is not really useful here.

Reply:
Actually my idea of a Mock database was to be able to use it everywhere where applicable, even here if it simplifies things.

However, I do not think that it would take much effort to actually store a list of jobs instead of the whole database.